src/main/java/net/sf/jabref/logic/msbib/BibTeXConverter.java

```diff
@@ -84,7 +83,12 @@ public static BibEntry convert(MSBibEntry entry) {
         }
         if (entry.month != null) {
             Month month = MonthUtil.getMonth(entry.month);
-            fieldValues.put(FieldName.MONTH, month.shortName);
+            //if we encouter an UnknownMonth/NULL_OBJECT month, the shortname returns null
+            if (month.shortName == null) {
```

Comment:
Yeah. Thought about remodeling the stuff. The NULL_OBJECT stuff and UnknownMonth is really shitty,